docs/ibc/integration.md

```diff
@@ -28,6 +28,16 @@ The first step is to add the following modules to the `BasicManager`: `x/capabil
 and `x/ibc-transfer`. After that, we need to grant `Minter` and `Burner` permissions to
 the `ibc-transfer` `ModuleAccount` to mint and burn relayed tokens.
 
+### Integrating Light Clients
+
+In order to integrate an external light client, you must take the following steps:
```

Comment:
I think this might be an opportunity to clarify that IBC core [includes these light clients](https://github.com/cosmos/ibc-go/tree/main/modules/light-clients) out of the box, but external light clients (ibc-go compatible) can also be added

Reply:
Do you recommend that the docs states the previous behaviour?

Reply:
I personally would note the difference with prior versions yes, imo better to repeat something once too many than potentially confuse a reader who has built up some expectations resulting from previous versions

Something like:
> Note that from v7 onwards, all light clients have to explicitly be registered in a chain's `app.go` and follow the steps listed below. This in contrast to earlier version of ibc-go when `07-tendermint` and `06-solomachine` were added out of the box.

This could be helpful for readers that are not necessarily working with production chains that are covered by the migration handler

Reply:
Great suggestions @colin-axner and @tmsdkeys I'll include these improvements 👍